src/components/_classes/component/Component.js

```diff
@@ -871,7 +871,7 @@ export default class Component extends Element {
     if (!template) {
       return '';
     }
-
+    data.t = this.t.bind(this);
```

Comment:
The ```t``` method is already added to the evalContext (I thought)...  as ```ctx.t()```.

The issue here is what happens if they have a field called "T" in their form. Won't this override that data with a function of ```t```?  This needs to change.

Reply:
In EditGrid to render the templates is used `renderString` method and it gives the error of `t` not defined, so I assigned it.
It will both work ok if I pass `t` in `evalContext` options.
Should I change it this way?

Reply:
Or I can pass it separately from `data` property in places where `renderString` is called in EditGrid component.

Reply:
Although, for now it is not actually overriding the data, but can be misleading. Will pass it from EditGrid.
![image](https://user-images.githubusercontent.com/54106407/102454104-7f29ba80-404e-11eb-8cf3-102bfe9bba95.png)